File: PharmacyApp/src/main/java/com/pharmacy/dto/ReplyDto.java
package com.pharmacy.dto;

public class ReplyDto {
	
	public String objectionId;
	public String content;
	
	public ReplyDto() {}
	
	public ReplyDto(String objectionId, String content) {
		super();
		this.objectionId = objectionId;
		this.content = content;
	}

	public String getObjectionId() {
		return objectionId;
	}

	public void setObjectionId(String objectionId) {
		this.objectionId = objectionId;
	}

	public String getContent() {
		return content;
	}

	public void setContent(String content) {
		this.content = content;
	}
	
	
	
}

File: PharmacyApplication/PharmacyApplication/Dto/ReplyDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PharmacyAPI.Dto
{
    public class ReplyDto
    { 
        public string ObjectionId { get; set; }
        public string Content { get; set; }

        public ReplyDto() { }

        public ReplyDto(string objectionId, string content)
        {
            ObjectionId = objectionId;
            Content = content;
        }
    }
}
